packages/core/src/providers/chainState/context.ts

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext } from 'react'
+import { ChainCall } from './callsReducer'
+import { ChainState } from './model'
+
+export const ChainStateContext = createContext<{
```

Comment:
Might not be required in this PR, but we should combine this with BlockNumber context

Reply:
I created https://github.com/EthWorks/useDapp/issues/23 to handle this separately.
There are two different block numbers, which is illustrated here:

https://github.com/EthWorks/dapp-boilerplate/blob/dapp-framework/packages/ui/src/components/page/BlockNumber.tsx#L11-L12